src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,44 @@
-export const TodoList = () => {};
+import { Todos } from '../../types/todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface TodoListProps {
+  todos: Todos;
+}
+
+export const TodoList: React.FC<TodoListProps> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
+        <TodoInfo todo={todo} key={todo.id} />
```

Comment:
Make sure that the `TodoInfo` component receives the necessary user data as props from the `TodoList` component. This requires changes in the `App` component to pass down the user data correctly.